meshroom/nodes/aliceVision/KeyframeSelection.py

```diff
@@ -341,5 +341,20 @@ class KeyframeSelection(desc.AVCommandLineNode):
             value=desc.Node.internalFolder,
             uid=[],
         ),
+        desc.File(
+            name="outputSfMDataKeyframes",
+            label="Output SfMData Keyframes File",
```

Comment:
The label seems a bit long compared to other similar labels in the UI. Maybe just `Keyframes SfMData` ?